array.go

```diff
@@ -1386,7 +1404,7 @@ func getMetadataValue(datatype Datatype, valueNum uint, cvalue unsafe.Pointer) (
 			value = *(*float64)(unsafe.Pointer(cvalue))
 		}
 	case TILEDB_CHAR:
-		tmpslice := (*[1 << 46]C.char)(unsafe.Pointer(cvalue))[:valueNum:valueNum]
+		tmpslice := (*[1 << 30]C.char)(unsafe.Pointer(cvalue))[:valueNum:valueNum]
```

Comment:
Is there a reason we can't use 46 for the size of the array here and are limited to 32 bit size?

Reply:
It does using 46 works too here and it seems that i still cannot reproduce the error

Reply:
It seems that the only way to have a segfault here is for `cvalue` to be `nil`

Reply:
Checked with vcf-sample-20/data array and reproduced the error. Indeed for index 4 `cvalue` is `null` pointer, `valueNum=1`. Can this be a bug in core?

Reply:
What is the metadata key for the value that is null?

Reply:
<img width="1140" alt="Screenshot 2020-07-14 at 12 35 57 AM" src="https://user-images.githubusercontent.com/2915496/87356125-3420ee00-c56a-11ea-986b-1c50cda035e0.png">